uint16[] id_list

uint16[] velocity_i_gain_pulse
uint16[] velocity_p_gain_pulse
uint16[] position_d_gain_pulse
uint16[] position_i_gain_pulse
uint16[] position_p_gain_pulse
uint16[] feedforward_2nd_gain_pulse
uint16[] feedforward_1st_gain_pulse